{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/settings.py

```diff
@@ -43,7 +43,7 @@
     '{{ cookiecutter.package_name }}',
 )
 
-if DEBUG:
+if DEBUG and config('DEBUG_TOOLBAR', default=True, cast=bool):
```

Comment:
It can be confusing to have to know that you need to set both DEBUG and DEBUG_TOOLBAR for it to show up. I'd think they are either completely independent or something like:

```
DEBUG = config('DEBUG', default=False, cast=bool)
DEBUG_TOOLBAR = config('DEBUG_TOOLBAR', default=DEBUG, cast=bool)

if DEBUG_TOOLBAR:
    ...
```